### DISCUSSION TITLE
Up to date guide for running on iOS

### DISCUSSION THREAD

**Discussion by ptliddle** [emilk/egui#5434] [Q&A]
I'm pretty new to using egui. I have a relatively simple egui app i wrote that's using eframe and i want to get it running on iOS. I'm looking for an up to date step by step guide on doing this and haven't been able to find one. If someone could point me in the right direction that would be great!

Thanks

**Comment by lucasmerlin** [comment]
I made an example using cargo mobile (2) a while ago, the setup should still work fine: https://github.com/lucasmerlin/egui-apple-example

**Comment by ptliddle** [comment]
Thanks. I tried this, but had trouble getting it to run.

I cloned the repo then tried doing `cargo mobile apple run`, also checked there was a valid target with `cargo mobile apple list` but i just get a error about no Config file. I'm still new to Rust so it's possible i'm doing something wrong.

`Didn't find "egui" template pack at "{redacted}/.cargo-mobile/templates/apps/egui.toml" or "{redacted}/.cargo-mobile/templates/apps/egui"`
